contrib/net/http/roundtripper.go

```diff
@@ -57,7 +61,7 @@ func (rt *roundTripper) RoundTrip(req *http.Request) (res *http.Response, err er
 		if rt.cfg.after != nil {
 			rt.cfg.after(res, span)
 		}
-		if rt.cfg.errCheck == nil || rt.cfg.errCheck(err) {
+		if !errors.Is(err, &events.BlockingSecurityEvent{}) && (rt.cfg.errCheck == nil || rt.cfg.errCheck(err)) {
```

Comment:
Nit: could this `&events.BlockingSecurityEvent{}` be declared as a var outside of `RoundTrip`? I think that it could save an allocation in each request.